src/components/molecules/EventDisplay/EventDisplay.scss

```diff
@@ -1,5 +1,5 @@
 $side-padding: 30px;
-$primary: #005ee5;
+$primary: #6f43ff;
 $black: #000000;
 $white: #fff;
```

Comment:
Since this is the same as the main definition in constants now, I think we can just delete this (and import constants if required)

```suggestion
```